src/script.js

```diff
@@ -441,6 +461,7 @@ module.exports = {
   isMultisigOutput: isMultisigOutput,
   isNullDataOutput: isNullDataOutput,
 
+  parseMultisigScript: parseMultisigScript,
```

Comment:
Yeah, it probably makes sense to do this for the scripts we support signing. While classifying we basically turn up the same information, so unifying that is good (avoids the need to call decompile a second time to get your script hash or whatever)